src/data/representative_content.py

```diff
@@ -0,0 +1,45 @@
+# Gets 1000 most 'representative' pieces of content for each top level taxon and it's children
+# Saves them to file
+
+import os
+import pandas as pd
+from sklearn.metrics import pairwise_distances_chunked
+from utils import directories as dirs
+
+class RepresentativeContent:
```

Comment:
Can't find any references to this code (except commented out in the run_all.py script). Still needed?

Reply:
Yes, it was commented out because it takes aaages to run